tests/playbooks/tests_ipv6_dns_search.yml

```diff
@@ -0,0 +1,93 @@
+---
+- hosts: all
+  tasks:
+    - import_role:
+        name: linux-system-roles.network
+      vars:
+        network_connections:
+          - name: br-example
+            type: bridge
+            ip:
+              dhcp4: false
+              auto6: false
+              dns: 8.8.8.8
+              dns_search: google.com
+              address:
+                - 10.2.2.77/26
+            state: up
+    - block:
+        - name: Verify nmcli connection DNS search entry for IPv4
+          shell: |
+            set -euxo pipefail
```

Comment:
why is this set command needed?

Reply:
https://github.com/linux-system-roles/network/pull/342#issuecomment-780137206

Reply:
I realized that there is no need to use the `shell` module, let's use `command` module instead.